src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/BulkCreationServiceTest.java

```diff
@@ -318,30 +318,54 @@ public void updateBulkRequestException() throws IOException {
+                + "midSearchCollection=null, multipleCollection=[MultipleTypeItem(id=0, "
+                + "value=MultipleType(caseIDM=0, ethosCaseReferenceM=1111, leadClaimantM=No, "
+                + "multipleReferenceM= , clerkRespM= , claimantSurnameM=Fernandez, respondentSurnameM=Mr Respondent, "
+                + "claimantRepM= , respondentRepM= , fileLocM= , "
+                + "receiptDateM= , positionTypeM= , feeGroupReferenceM=111122211, jurCodesCollectionM= , "
+                + "stateM=Accepted, subMultipleM= , subMultipleTitleM= , "
+                + "currentPositionM= , claimantAddressLine1M= , claimantPostCodeM= , respondentAddressLine1M= , "
+                + "respondentPostCodeM= , flag1M= , flag2M= , EQPM= , " +
+                "respondentRepOrgM= , claimantRepOrgM= )), MultipleTypeItem(id=0, value=MultipleType(caseIDM=0," +
+                " ethosCaseReferenceM=111111, leadClaimantM=No, " +
+                "multipleReferenceM= , clerkRespM= , claimantSurnameM=Fernandez, respondentSurnameM=Mr Respondent, " +
+                "claimantRepM= , respondentRepM= , fileLocM= , " +
+                "receiptDateM= , positionTypeM= , feeGroupReferenceM=111122211, jurCodesCollectionM= , stateM=Accepted, " +
+                "subMultipleM= , subMultipleTitleM= , " +
+                "currentPositionM= , claimantAddressLine1M= , claimantPostCodeM= , respondentAddressLine1M= ," +
+                " respondentPostCodeM= , flag1M= , flag2M= , EQPM= , " +
+                "respondentRepOrgM= , claimantRepOrgM= )), MultipleTypeItem(id=0, value=MultipleType(caseIDM=0, " +
+                "ethosCaseReferenceM=1122, leadClaimantM=No, " +
+                "multipleReferenceM= , clerkRespM= , claimantSurnameM=Fernandez, respondentSurnameM=Mr Respondent, " +
+                "claimantRepM= , respondentRepM= , fileLocM= , " +
+                "receiptDateM= , positionTypeM= , feeGroupReferenceM=111122211, jurCodesCollectionM= , stateM=Accepted," +
+                " subMultipleM= , subMultipleTitleM= , " +
+                "currentPositionM= , claimantAddressLine1M= , claimantPostCodeM= , respondentAddressLine1M= , " +
+                "respondentPostCodeM= , flag1M= , flag2M= , EQPM= , " +
+                "respondentRepOrgM= , claimantRepOrgM= ))], subMultipleCollection=null, subMultipleDynamicList=null," +
+                " searchCollectionCount=null, " +
+                "multipleCollectionCount=3, correspondenceType=null, correspondenceScotType=null, selectAll=null, " +
+                "scheduleDocName=null, positionType=null, flag1=null, " +
+                "flag2=null, EQP=null, submissionRef=null, claimantOrg=null, respondentOrg=null, state=null, " +
+                "flag1Update=null, flag2Update=null, EQPUpdate=null, " +
+                "jurCodesDynamicList=null, outcomeUpdate=null, filterCases=null, docMarkUp=null," +
+                " multipleSource=Manually Created))";
```

Comment:
Better held as a constant to not make actual test messy.
Also, public not required for unit tests

Reply:
using constant now, however can't remove public, the test in not found/run if I remove public keyword